packages/angular/build/src/builders/application/schema.json

```diff
@@ -37,6 +37,33 @@
       "type": "string",
       "description": "Customize the base path for the URLs of resources in 'index.html' and component stylesheets. This option is only necessary for specific deployment scenarios, such as with Angular Elements or when utilizing different CDN locations."
     },
+    "security": {
+      "description": "Security features to protect against XSS and other common attacks",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "autoCsp": {
+          "description": "Enables auto-CSP generation. Will default to true once we are out of experimental/preview phases.",
```

Comment:
Suggestion: Some users might not immediately know what "CSP" means in this context, so the description should probably spell out the term and give a little more details. Maybe say "Enables automatic generation of a Content Security Policy based on scripts in `index.html`." If there's a good MDN or Chrome Developers link explaining what CSP is, that would also be good to link here.

Once the documentation for this feature exists on angular.dev, it would be great to update this description to point to that documentation and help users understand exactly what "auto-CSP generation" actually means.

Reply:
Updated the description and added a link to web.dev that describes our approach-- IIRC the MDN documentation around strict hash-based CSPs are WIP right now, but we'd be super happy to swap to an angular.dev or a MDN source once we have a canonical source there to point to. Thanks!